rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+
+from jina.executors.rankers import Chunk2DocRanker
+import pytest
+from .. import SimpleAggregateRanker
+
+
+def chunk_scores(is_positive_score=True):
```

Comment:
is the `is_positive_score` ever changed in this test?

Reply:
yes I had one test case where it was set to False. 
Now I use the `factor` parameter to define the negative number case (-1), positive case (1) and zero case (0).